django-stubs/forms/utils.pyi

```diff
@@ -26,8 +26,16 @@ class RenderableMixin:
         context: dict[str, Any] | None = ...,
         renderer: BaseRenderer | type[BaseRenderer] | None = ...,
     ) -> SafeString: ...
-    __str__ = render
-    __html__ = render
+
+    __str__: Callable[..., SafeString] = ...
+    __html__: Callable[..., SafeString] = ...
+
+class RenderableFieldMixin(RenderableMixin):
+    def as_field_group(self) -> SafeString: ...
+    def as_hidden(self) -> SafeString: ...
+    def as_widget(self) -> SafeString: ...
+    __str__: Callable[..., SafeString]
```

Comment:
We can remove `__str__` and `__html__` here, because this class extends `RenderableMixin`

Reply:
@sobolevn  Failure to add to this code, including the code above, will result in the following error.


error: not checking stubs due to mypy build errors:
django-stubs/forms/boundfield.pyi:44: error: Signature of "__html__" incompatible with supertype "RenderableMixin"  [override]
django-stubs/forms/boundfield.pyi:44: note:      Superclass:
django-stubs/forms/boundfield.pyi:44: note:          def render(self: RenderableMixin, template_name: Optional[str] = ..., context: Optional[Dict[str, Any]] = ..., renderer: Union[BaseRenderer, Type[BaseRenderer], None] = ...) -> SafeString
django-stubs/forms/boundfield.pyi:44: note:      Subclass:
django-stubs/forms/boundfield.pyi:44: note:          def __html__(self) -> SafeString
sajaehyeog-ui-MacBookAir:django-stubs sajh$ bash ./scripts/stubtest.sh